File-Formats/DocIO/Word-document/Iterating-Word-document-elements.md

```diff
@@ -1059,4 +1059,8 @@ private static void IterateParagraph(ParagraphItemCollection paraItems)
 
 {% endtabs %}
 
-You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Word-document/Iterate-document-elements).
\ No newline at end of file
+You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Word-document/Iterate-document-elements).
+
+T> Passing null for both the property names and property values in above APIs, you can also find an item in a Word document without relying on any property.
```

Comment:
Please move this tip to Find an item page

Reply:
Addressed